fiftyone/operators/store/service.py

```diff
@@ -47,10 +47,12 @@ def __init__(
             repo = RepositoryFactory.execution_store_repo(
                 dataset_id=dataset_id
             )
-
+        self._dataset_id = dataset_id
         self._repo: ExecutionStoreRepo = repo
 
-    def create_store(self, store_name: str) -> StoreDocument:
+    def create_store(
```

Comment:
Same `creat_store` comment as before, I don't care which method name is used - I added a try/except for both just in case in DQ panel